x-pack/test_serverless/api_integration/test_suites/common/index_management/indices.ts

```diff
@@ -5,33 +5,37 @@
  * 2.0.
  */
 
-import expect from 'expect';
+import expect from '@kbn/expect';
 import { FtrProviderContext } from '../../../ftr_provider_context';
 
-const API_BASE_PATH = '/api/index_management';
 const INTERNAL_API_BASE_PATH = '/internal/index_management';
 const expectedKeys = ['aliases', 'hidden', 'isFrozen', 'primary', 'replica', 'name'].sort();
 
 export default function ({ getService }: FtrProviderContext) {
   const supertest = getService('supertest');
   const es = getService('es');
   const log = getService('log');
+  const indexManagementService = getService('indexManagement');
 
   describe('Indices', function () {
-    const indexName = `index-${Math.random()}`;
+    let indexName: string;
+    let reload: typeof indexManagementService['indices']['api']['reload'];
+    let list: typeof indexManagementService['indices']['api']['list'];
+    let deleteIndex: typeof indexManagementService['indices']['api']['deleteIndex'];
+    let createIndex: typeof indexManagementService['indices']['helpers']['createIndex'];
+    let deleteAllIndices: typeof indexManagementService['indices']['helpers']['deleteAllIndices'];
+    let catIndex: typeof indexManagementService['indices']['helpers']['catIndex'];
 
     before(async () => {
-      // Create a new index to test against
-      const indexExists = await es.indices.exists({ index: indexName });
-
-      // Index should not exist, but in the case that it already does, we bypass the create request
```

Comment:
We don't need this check anymore, since an index is being created with a random index name